converter/internal/promtailconvert/internal/build/stages.go

```diff
@@ -255,12 +260,17 @@ func convertDrop(cfg interface{}, diags *diag.Diagnostics) (stages.StageConfig,
 
 	var longerThan units.Base2Bytes
 	if pDrop.LongerThan != nil {
-		lt, err := units.ParseBase2Bytes(*pDrop.LongerThan)
+		var pLongerThan flagext.ByteSize
```

Comment:
Promtail and Flow use different libraries for parsing byte sizes from strings. 
So we read using promtail's library and then write using flow's library.